src/pages/[marketplace]/deals/index.tsx

```diff
@@ -112,7 +113,7 @@ const Deals: NextPageWithLayout = () => {
 	const mapDeal = useCallback(
 		({ address, name, principal, goLiveAt, interestRepaid, totalInterest, borrower }: Deal) => {
 			const isRepayable = borrower.toString() === publicKey?.toString();
-			const path = `/${marketplace}/deals/${address.toString()}/${isRepayable ? "repay" : "show"}`;
+			const path = `/${marketplace}/deals/${isRepayable ? "repay" : "show"}?did=${address.toString()}`;
```

Comment:
did is kind of the accepted abbreviation for decentralized identity but not the biggest issue rn

Reply:
I'll make a ticket. It's just something I took from the nextjs docs, but wasn't the biggest fan of that name anyway.